OpenCaseManager/Scripts/journalNote.js

```diff
@@ -132,7 +136,7 @@ function formatDate(_date) {
 
     var day = value.getDate() < 10 ? '0' + value.getDate() : value.getDate();
     var month = value.getMonth + 1;
-        month = value.getMonth() < 10 ? "0" + value.getMonth() : value.getMonth();
+        month = (value.getMonth()+1) < 10 ? "0" + (value.getMonth()+1) : (value.getMonth()+1);
```

Comment:
`+1` kan nok godt placreres et andet sted så vi ikke skal gøre det hver gang, men for nu må det vist være fint nok